### ISSUE TITLE
PropTypes are not longer under React class

### ISSUE THREAD

**Issue by rubelux** [TinRod/react-plaid#1]
```
TypeError: Cannot read property 'bool' of undefined
./node_modules/react-plaid/lib/index.js
node_modules/react-plaid/lib/index.js:121
  118 | };
  119 | 
  120 | ReactPlaid.propTypes = {
> 121 |   open: _react.PropTypes.bool,
  122 |   // Displayed once a user has successfully linked their account
  123 |   clientName: _react.PropTypes.string.isRequired,
  124 | 
```

**Comment by browniefed** [comment]
Do you actually use this?

**Comment by browniefed** [comment]
Published  react-plaid@0.2.0 with fixes.
